custom-metrics-stackdriver-adapter/pkg/adapter/translator/query_builder.go

```diff
@@ -512,6 +520,14 @@ func (t *Translator) ListMetricDescriptors(fallbackForContainerMetrics bool) *st
 // GetMetricKind returns metricKind for metric metricName, obtained from Stackdriver Monitoring API.
 func (t *Translator) GetMetricKind(metricName string, metricSelector labels.Selector) (string, string, error) {
 	metricProj := t.config.Project
+	cacheKey := metricKindCacheKey{
```

Comment:
Could you add a test for GetMetricKind in query_builder_test.go that covers both cache hit and miss?